docs/community/faq/faq-users.md

```diff
@@ -2,69 +2,85 @@
+   - Yes, `setup()` has an `n_jobs` parameter which allows to parallelise cross-validation and hyperparameter optimisation.
 
-3. Can AutoEmulate be integrated with other data analysis or simulation tools?
+2. Can AutoEmulate be integrated with other data analysis or simulation tools?
    <!-- Information on APIs, file formats, or protocols that facilitate the integration of AutoEmulate with other software ecosystems. -->
+   - `AutoEmulate` takes simple `X` and `y` ndarrays as input, and returns emulator models that can be saved and loaded with `joblib`. All emulators are written as scikit learn estimators, so they can be used like any other scikit learn model in a pipeline.
 
 ## Data Handling
 
-1. What are the best practices for data preprocessing before using AutoEmulate?
+1. What are the best practices for data preprocessing before using `AutoEmulate`?
    <!-- Tips and recommendations on preparing data, including normalisation, dealing with missing values, or data segmentation. -->
+   - The user will typically run their simulation on a selected set of input parameters (-> experimental design) using a latin hypercube or other sampling method. `AutoEmulate` currently needs all inputs to be numeric and we don't support missing data. By default, `AutoEmulate` will scale the input data to zero mean and unit variance, and there's the option to do dimensionality reduction in `setup()`.
 
 2. How does AutoEmulate handle large datasets?
    <!-- Advice on managing large-scale data analyses, potential memory management features, or ways to streamline processing. -->
-
-3. Can I use AutoEmulate for real-time data analysis?
-   <!-- Insights into the software's ability to process data in real-time and any limitations or considerations. -->
+   - `AutoEmulate` is optimised to work with smaller datasets (in the order of hundreds to thousands of samples). Training emulators with large datasets (hundreds of thousands of samples) may currently require a long time and is not recommended. Emulators are created because it's expensive to evaluate the simulation, so we expect most users to have a relatively small dataset.
 
 ## Troubleshooting
 
-1. What common issues might I encounter when using AutoEmulate, and how can I solve them?
+1. What common issues might I encounter when using `AutoEmulate`, and how can I solve them?
    <!-- A list of frequently encountered problems with suggested solutions, possibly linked to a more extensive troubleshooting guide. -->
+   - TODO
 
-2. How can I report a bug or request a feature in AutoEmulate?
+2. How can I report a bug or request a feature in `AutoEmulate`?
    <!-- Instructions on the proper channels for reporting issues or suggesting enhancements, including any templates or information to include. -->
+   - Please open an issue using the [bug issue template](https://github.com/alan-turing-institute/autoemulate/issues/new/choose).
```

Comment:
This actually links to the page where the user can open a bug issue report OR request a new feature, so good to have the link here but suggestion to write something like (this isn't great so feel free to edit): "You can report a bug or request new features through the issue templates in our GitHub repository. Head on over there and choose one of the templates for your purpose and get started."

Reply:
Yes, agree, changed the text to your suggestion!